### PR TITLE
Fix image size quote and thumbnail hidden issue on H5P

### PR DESCRIPTION
**Pull request by simonebelotti75**

### CODE REVIEW

**Review by muhammadqamar — DISMISSED** [review]

File: src/components/ResourceCard/EditResource/index.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const EditResource = (props) => {
 
   useEffect(() => {
     showResourceDescribeActivity(resource, match.params.activityId);
-  }, []);
+  }, [match.params.activityId, resource, showResourceDescribeActivity]);
```
Comment (simonebelotti75) [inline src/components/ResourceCard/EditResource/index.js:29]: I think I removed resource here
Did you add it?

File: src/components/models/GoogleLoginModal.js [hunk 2]
```diff
@@ -138,9 +142,9 @@ const GoogleLoginModal = ({
                         placeholder="Course Name"
                       >
                         <option>Select your room</option>
-                        {rooms.map((data) => {
-                          return <option>{data}</option>;
-                        })}
+                        {rooms.map((data) => (
+                          <option>{data}</option>
```
Comment (simonebelotti75) [inline src/components/models/GoogleLoginModal.js:146]: I didn't notice before but I think we need key attribute here
Do you know which property of data we can use as unique key?

File: src/containers/Projects/CreateProjectPopup/index.js [hunk 3]
```diff
@@ -286,9 +287,7 @@ let CreateProjectPopup = (props) => {
 
               <div
                 className="gallery"
-                onClick={() => {
-                  openFile.current.click();
-                }}
+                onClick={openFile.current.click}
```
Comment (simonebelotti75) [inline src/containers/Projects/CreateProjectPopup/index.js:290]: I think I reverted it on my commit
It was throwing error on page refresh when I change in this way

**Review by simonebelotti75 — COMMENTED** [review]
Please check comments I wrote

File: src/components/ResourceCard/EditResource/index.js (hunk 1 above)
Comment (muhammadqamar) [reply src/components/ResourceCard/EditResource/index.js:29]: no i didn't, it should not have any dependency

Comment (simonebelotti75) [reply src/components/ResourceCard/EditResource/index.js:29]: It should have those but not resource

Comment (simonebelotti75) [reply src/components/ResourceCard/EditResource/index.js:29]: I will update it

File: src/components/models/GoogleLoginModal.js (hunk 2 above)
Comment (muhammadqamar) [reply src/components/models/GoogleLoginModal.js:146]: u can use data.id

File: src/containers/Projects/CreateProjectPopup/index.js (hunk 3 above)
Comment (muhammadqamar) [reply src/containers/Projects/CreateProjectPopup/index.js:290]: kindly check on yr local and revert this change if it is breaking

**Review by muhammadqamar — APPROVED** [review]

**Pull request merged**